content/applications/marketing/marketing_automation/advanced/understanding_metrics.rst

```diff
@@ -2,79 +2,84 @@
 Understanding metrics
 =====================
 
-Metrics are detailed statistics that measure the progress (and success) of marketing campaigns.
-When creating marketing activities in a workflow, Odoo will visually display various metrics
-related to the campaign with graphs and data.
+Metrics are detailed statistics that measure the progress (and success) of marketing campaigns. When
+creating marketing activities in a workflow, Odoo visually displays various metrics related to the
+campaign with detailed graphs and in-depth data.
 
 Consider the following example:
 
 .. image:: understanding_metrics/commercial-prospection-campaign-example.png
    :align: center
    :alt: An example of a marketing automation campaign in Odoo.
 
-In this case, the :guilabel:`Target` of this campaign is set to :guilabel:`Lead/Opportunity` and
-was narrowed down to only focus on the Leads (or Opportunities) whose :guilabel:`Tag Name` contain
-a value of `product` and they *also* have an email address set up in the database.
+In this case, the :guilabel:`Target` of this campaign is set to :guilabel:`Lead/Opportunity`, and
+was narrowed down to *only* focus on the leads (or opportunities) whose :guilabel:`Tag Name`
+contains a value of `product` - and *also* have an email address set up in the database.
 
-Beneath the filter nodes, Odoo will display how many records in the database fall into the
-previously-specified criteria filter. In this example, there are 18 records in the database that
-match the criteria.
+Beneath the filter nodes, Odoo displays how many records in the database fall into the
+previously-specified criteria filter.
+
+In this example, there are 18 records in the database that match the criteria.
```

Comment:
This seems more appropriate in a supplemental **example** admonition block. The documentation cannot be one long use case that heavily depends on imagery to communicate the knowledge.